etc/profile-m-z/ncmpcpp.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for ncmpcpp
+# Description: Featureful ncurses based MPD client inspired by ncmpc
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ncmpcpp.local
+# Persistent global definitions
+include globals.local
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+blacklist ${RUNUSER}
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+#include disable-write-mnt.inc
+include disable-X11.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/ncmpcpp
+mkdir ${HOME}/.lyrics
+whitelist ${HOME}/.config/ncmpcpp
+whitelist ${HOME}/.lyrics
```

Comment:
Add these paths to disable-programs.inc and add corresponding `noblacklist`
entries in this profile.

This should be done whenever adding program-specific paths.

Also, if any of the configuration files allow arbitrary code execution, add
`read-only $path` to disable-common.inc (and `read-write $path` to this profile
if the configuration file can be edited by the program itself).

This also applies to the other PRs.

Reply:
There are a lot of suggested changes. I will try to apply all changes at once tomorrow.